kornia/augmentation/_2d/intensity/planckian_jitter.py

```diff
@@ -168,7 +169,8 @@ def apply_transform(
         self, input: Tensor, params: Dict[str, Tensor], flags: Dict[str, Any], transform: Optional[Tensor] = None
     ) -> Tensor:
         list_idx = params["idx"].tolist()
-
+        KORNIA_CHECK_SHAPE(input, ["*", "3", "H", "W"])
```

Comment:
do you want me to split this PR?

one for this fix, one for the docs, other for the benchmarks itself

Reply:
Yeah, no prob